kolibri/utils/options.py

```diff
@@ -246,6 +267,71 @@ def multiprocess_bool(value):
         return False
 
 
+class LazyImportFunction(object):
```

Comment:
isn't this the same purpose as the builtin `importlib.util.LazyLoader` class? I know this is not available in 2.7 but just wondering if we could hijack its code for 2.7 and use the original one  for > 3.5 https://github.com/python/cpython/blob/main/Lib/importlib/util.py#L268

If it's not adecuate, I think we should have at least some test case for this as it's a quite sofisticated piece of code (first time I see a real case use for AST out of parsers world)

Reply:
Yes - it does seem better to vendor this, I was just totally unaware of it.